Documentation/README_TapToPlace.md

```diff
@@ -0,0 +1,95 @@
+    > [!NOTE]
+    > Tap to Place is a stand alone Solver and cannot be chained with other solvers. 
+    - Tap to Place Properties:
+        - `Auto Start`: If true, the game object to place will start out selected.  The object will immediately start following the TrackedTargetType (Head or Controller Ray). This value must be modified before Start() is invoked in order to have any effect.
+        - `Default Placement Distance`: The default distance (in meters) an object will be placed relative to the TrackedTargetType forward in the SolverHandler. The game object will be placed at the default placement distance if a surface is not hit by the raycast.
+        - `Max Raycast Distance`: The max distance (meters) for the raycast based on the 'TrackedTargetType' origin. This raycast looks for a surface to place a selected object.
+        - `Surface Normal Offset`: The distance between the center of the game object to place and a surface along the surface normal, if the raycast hits a surface. By default, the surface normal offset is 0 but after Start() the surface normal offset is set to the collider's extents along the z axis.
+        - `Keep Orientation Vertical`: If true, the game object to place will remain upright and in line with Vector3.up.
+        - `Rotate According to Surface`: If false, the game object to place will not change its rotation according to the surface hit.  The object will remain facing the camera while IsBeingPlaced is true.
+        - `Magnetic Surfaces`: An array of LayerMasks to execute from highest to lowest priority. First layer mask to provide a raycast hit will be used for position calculations.
+        - `Debug Enabled`: If true and in the Unity Editor, the normal of the raycast hit will be drawn in yellow. Debug enabled is useful when `RotateAccordingToSurface` is true because it draws the normal of the surface hit which visually explains why the object is set at its current orientation.
+        - `On Placing Started`: This event is triggered once when the game object to place is selected.
+        - `On Placing Stopped`: This event is triggered once when the game object to place is unselected, placed.
+
+1.  Testing Tap to Place behavior in editor
+    - Press play and hold the space bar to show an input simulation hand.
+    - Move the hand until the cube is in focus and simulate a click with the input simulation hand by clicking with the left mouse.
+        - If colliders are not present in the scene, the object will follow the `TrackedTargetType` at the defined `Default Placement Distance`.
+    - The object will follow the movement of the `TrackedTargetType` after selection. To simulate head movement in editor, press the WASD keys. Change head rotation by clicking and holding the right mouse.
```

Comment:
I am tempted to say remove the WASD keys/controlling head movement. Maybe just reference the input simulation page. If things change or the user has changed their project, this just seems another area that would have to be updated and may be invalidated.

Reply:
I think I am going to keep it in there for now since this documentation is in the form of a tutorial and I like having all the information on one page.